packages/sdk/client-services/src/packlets/identity/identity-manager.ts

```diff
@@ -137,15 +157,7 @@ export class IdentityManager {
       credentials.push(await generator.createDeviceAuthorization(identityRecord.deviceKey));
 
       // Write device metadata to profile.
-      credentials.push(
-        await generator.createDeviceProfile({
-          platform: platform.name,
-          platformVersion: platform.version,
-          architecture: typeof platform.os?.architecture === 'number' ? String(platform.os.architecture) : undefined,
-          os: platform.os?.family,
-          osVersion: platform.os?.version,
-        }),
-      );
+      credentials.push(await generator.createDeviceProfile(deviceProfile));
```

Comment:
We need to add similar credentials in the method `IdentityManager.acceptIdentity`